src/database/schema.sql

```diff
@@ -229,7 +231,7 @@ CREATE TABLE Likes (
 	FOREIGN KEY (listing_id) REFERENCES Listings ON DELETE CASCADE
 );
 
-CREATE TABLE OrganisationLikes (
+CREATE TABLE organisationlikes (
```

Comment:
change this to singular `organisationlike` as this will be our db table naming convention moving forward